Presentation.Web/Controllers/API/V2/Internal/GdprExportReportInternalV2Controller.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Core.Abstractions.Types;
+using Core.ApplicationServices.SystemUsage.GDPR;
+using Core.DomainModel.ItSystemUsage.GDPR;
+using Core.DomainModel.Organization;
+using Core.DomainServices.Generic;
+using Presentation.Web.Infrastructure.Attributes;
+using Presentation.Web.Models.API.V2.Internal.Response;
+using Swashbuckle.Swagger.Annotations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Web.Http;
+using Presentation.Web.Controllers.API.V2.External.ItSystemUsages.Mapping;
+
+namespace Presentation.Web.Controllers.API.V2.Internal
+{
+     
+    [RoutePrefix("api/v2/internal/gdpr-report")]
+    public class GdprExportReportInternalV2Controller : InternalApiV2Controller
+    {
+        private readonly IGDPRExportService _gdprExportService;
+        private readonly IEntityIdentityResolver _entityIdentityResolver;
+
+        public GdprExportReportInternalV2Controller(IGDPRExportService gdprExportService, IEntityIdentityResolver entityIdentityResolver)
+        {
+            _gdprExportService = gdprExportService;
+            _entityIdentityResolver = entityIdentityResolver;
+        }
+
+        [HttpGet]
+        [Route("{organizationUuid}")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(IEnumerable<GdprReportResponseDTO>))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        public IHttpActionResult GetGdprReport([FromUri][NonEmptyGuid] Guid organizationUuid)
+        {
+            return _entityIdentityResolver.ResolveDbId<Organization>(organizationUuid)
```

Comment:
It's a small detail, but this orchestration of services and selecting operation errors could go to an application service so the controller just handles http requests.